tests/src/system/basic/WskBasicTests.scala

```diff
@@ -348,14 +348,14 @@ class WskBasicTests
             assetHelper.withCleaner(wsk.rule, ruleName) {
                 (rule, name) =>
                     rule.create(name, trigger = triggerName, action = actionName)
-                    rule.create(name, trigger = triggerName, action = actionName, update = true)
+                    rule.create(name, trigger = triggerName, action = actionName, update = true, expectedExitCode = CONFLICT)
```

Comment:
now that rules are enabled by default, the second "update" operation will result in a CONFLICT. we have a couple of options here.

1. as i did, perform the update and expect a conflict. this will check that the first (creation) operation indeed enabled the rule

2. add a disable call in-between, and revert the update line to expect success

thoughts?